Modules/Core/Controllers/CustomFieldsController.php

```
@@ -13,39 +20,82 @@ public function __construct(CustomFieldService $customFieldService)
 
+    /**
+     * Display form for creating or editing a custom field.
+     *
+     * @param int|null $id Custom field ID (null for create)
+     *
+     * @return \Illuminate\View\View|\Illuminate\Http\RedirectResponse
+     *
+     * @legacy-function form
+     * @legacy-file application/modules/custom_fields/controllers/Custom_fields.php
+     */
     public function form(?int $id = null)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)